### DISCUSSION TITLE
Error running compensation notebook 4a

### DISCUSSION THREAD

**Discussion by spc27** [angelolab/toffy#462] [Q&A] [answered]
Hello,
I'm trying to test out the rosetta compensation using the sample FOVs that are available in toffy. I keep getting into the same error when running section [109] where I want to run the compensation with the three multipliers. Can you guys please give me some insights? In this particular error I set current_channel_name = 'CD3e' I saw someone saying that we should put the mass number instead of the channel name but this didn't work for me either. Would really appreciate your help!
<img width="956" alt="Screenshot 2024-05-19 at 17 11 19" src="https://github.com/angelolab/toffy/assets/117159668/877333be-8aee-4ea7-a51e-fcb690fdb57f">

**Comment by spc27** [comment, marked as answer]
Solved it already by matching the masses from the panels with the masses from the commercial rosetta matrix

**Comment by kevin53L** [comment]
hello, 

I am having the same error, can you give a little more insight on how you fixed the issue? More specifically how did you match the masses?

**Comment by spc27** [comment]
Hi,
What you need to do is to make sure that all the masses that are present in your panel file matches the rosetta matrix masses (both in the columns and rows). Apart from that, make sure that you have the image of all the channels in your panel file/rosetta matrix. Hope it helps
